manifests/test-app-conjur-authenticator-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: RoleBinding
+apiVersion: v1
+metadata:
+  name: test-app-conjur-authenticator-role-binding
+  namespace: {{ TEST_APP_PROJECT_NAME }}
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: {{ CONJUR_PROJECT_NAME }}
+roleRef:
+  kind: ClusterRole
+  name: pod-get-list-exec-test
```

Comment:
this should be `conjur-authenticator` to reference the ClusterRole created in the deployment scripts

Reply:
Ah right good catch. Thanks.